warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java

```diff
@@ -249,7 +249,7 @@ protected void printUsage() {
 
     @Override
     public int run(String[] args) throws Exception {
-        long setupStart = System.currentTimeMillis();
+        long setupstart = System.currentTimeMillis();
```

Comment:
Why change? Do you not like Camel Case?

Reply:
I was being consistent with the setupstop variable.  I had to choose 1 due to a merge conflict.  I can change them all to camel case if you care that much.

Reply:
I will change them